src/portal/src/i18n/lang/en-us-lang.json

```diff
@@ -428,7 +428,12 @@
         "NAME_REQUIRED": "Name is required",
         "NOTIFY_TYPE": "Notify Type",
         "EVENT_TYPE": "Event Type",
-        "EVENT_TYPE_REQUIRED": "Require at least one event type"
+        "EVENT_TYPE_REQUIRED": "Require at least one event type",
+        "PAYLOAD_FORMAT": "Payload Format",
```

Comment:
The term "Payload Format" is very broad in its definition.  I suggest changing it to the correct existing term **Message Format**

Reply:
Discussed with @wy65701436 and @chlins, and they believe `Payload Format` is better(Refer to https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads)